src/vue/components/nav/navbuttons-large.vue

```diff
@@ -8,7 +8,7 @@
       </div>
     </alph-tooltip>
 
-    <alph-tooltip :tooltipText="l10n.getText('TOOLTIP_SHOW_DEFINITIONS')" tooltipDirection="bottom-narrow" v-show="$store.getters['app/defDataReady']">
+    <alph-tooltip :tooltipText="l10n.getText('TOOLTIP_SHOW_DEFINITIONS')" tooltipDirection="bottom-narrow" v-show="$store.getters['app/fullDefDataReady']">
```

Comment:
I think, that we publish here short defs too, may be both checks?

Reply:
see previous comment